bitnami/external-dns/templates/secret.yaml

```diff
@@ -45,7 +45,7 @@ data:
   infoblox_wapi_password: {{ .Values.infoblox.wapiPassword | b64enc | quote }}
   {{- end }}
   {{- if eq .Values.provider "pdns" }}
-  pdns_api_key: {{ .Values.pdns.apiKey | b64enc | quote }}
+  pdns_api_key: {{ include "external-dns.pdns-api-key" . | b64enc | quote }}
```

Comment:
Where is this helper defined? I don't see it

Reply:
Ups, a typo